### PR TITLE
app-containers/lxd: add 5.18

### PR DESCRIPTION
**Pull request by listout**

### CODE REVIEW

**Comment by listout** [comment]
(performing tests)

**Comment by listout** [comment]
We should probably find better way of handling the lxd-5.18-remove-shellcheck-buildsystem-checks patch. I fear it can quickly become annoying and eat up space in tree.

**Review by juippis — COMMENTED** [review]
The patches aren't too big to be an issue, I'd more be annoyed having to rebase every time a new release is made. I thought it has been updated upstream already, too? Does it really keep printing errors if you don't include the patch at all?

Related, are we also affected by:
https://github.com/canonical/lxd/issues/12295
https://github.com/canonical/lxd/pull/12294

**Comment by listout** [comment]
> Does it really keep printing errors if you don't include the patch at all?

**Comment by juippis** [comment]
> 
> Shall I go ahead and make patch from [canonical/lxd@2d84817](https://github.com/canonical/lxd/commit/2d8481712055d0cbb3bd952e25a1df469939990d) to carry downstream?

I'm not sure if that is snap-related, or if we're affected too. But since this isn't merged yet it's safer to include it yeah.

File: app-containers/lxd/lxd-5.18.ebuild [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+VERIFY_SIG_OPENPGP_KEY_PATH=${BROOT}/usr/share/openpgp-keys/canonical.asc
+
+# The testsuite must be run as root.
+# make: *** [Makefile:156: check] Error 1
+RESTRICT="test"
+
+GOPATH="${S}/_dist"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-remove-shellcheck-buildsystem-checks.patch
```
Comment (juippis) [inline app-containers/lxd/lxd-5.18.ebuild:85]: You forgot to remove this patch, no need to push the *.patch* file either.

**Review by juippis — APPROVED** [review]
Thanks! I think I'll remove proxy-maint from metadata for this package, since it should be pretty covered even without that project. To reduce noise.

**Pull request closed without merging**